server/src/main/java/com/cloud/user/PasswordPolicy.java

```diff
@@ -79,8 +79,8 @@ public interface PasswordPolicy {
             "Advanced",
             String.class,
             "password.policy.regex",
-            ".+",
-            "A regular expression that the user's password must match. The default expression '.+' will match with any password.",
+            "",
+            "A regular expression that the user's password must match. By default no expression is used.",
```

Comment:
@JoaoJandre , this means that empty passwords are allowed, right? If so, do we think this is ok?

Reply:
@DaanHoogland 
password is a required field when create a user, it is not empty when create user via API or UI.
but it is empty when import a ldap user. we need to support both scenarios.

@JoaoJandre 
can we just skip the check if password is empty ?
```
     public void verifyIfPasswordCompliesWithPasswordPolicies(String password, String username, Long domainId) {
+        if (StringUtils.isEmpty(password)) {
+            return; // password is empty when import LDAP user
+        }
         int numberOfSpecialCharactersInPassword = 0;
         int numberOfUppercaseLettersInPassword = 0;
         int numberOfLowercaseLettersInPassword = 0;
```